src/test/java/io/stargate/sgv2/jsonapi/api/v1/CreateCollectionIntegrationTest.java

```diff
@@ -269,74 +268,6 @@ public void duplicateVectorCollectionNameWithDiffSetting() {
     }
   }
 
-  @Nested
```

Comment:
Moved to separate class to be able to use different system property overrides for passing/failing tests.